project/plugins.sbt

```diff
@@ -5,3 +5,7 @@ addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.3")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.3.5")
 
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.1.0")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.0")
```

Comment:
Any argument for why we should include `sbt-updates` inside the project? By default, I would guess this is better left as a global plugin under `~/.sbt`, since there's nothing inside the project using the plugin.